crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -327,25 +326,22 @@ async fn test_publish_gas() -> anyhow::Result<()> {
     )?;
     gas_status.charge_storage_read(gas_object.object_size_for_gas_metering())?;
     gas_status.charge_publish_package(publish_bytes.iter().map(|v| v.len()).sum())?;
-    gas_status.charge_storage_mutation(0, package.object_size_for_gas_metering(), 0.into())?;
+    gas_status.charge_storage_mutation(package.object_size_for_gas_metering(), 0.into())?;
     // Remember the gas used so far. We will use this to create another failure case latter.
     let gas_used_after_package_creation = gas_status.summary().gas_used();
-    gas_status.charge_storage_mutation(
-        gas_object.object_size_for_gas_metering(),
-        gas_object.object_size_for_gas_metering(),
-        0.into(),
-    )?;
+    gas_status.charge_storage_mutation(gas_object.object_size_for_gas_metering(), 0.into())?;
     // Actual gas cost will be greater than the expected summary because of the cost to discard the
     // `UpgradeCap`.
     let gas_summary = gas_status.summary();
     assert!(gas_cost.computation_cost >= gas_summary.computation_cost);
     assert_eq!(gas_cost.storage_cost, gas_summary.storage_cost);
+    println!("gas cost was {:?}", gas_cost);
```

Comment:
This is a draft, but just noting that these can go away.

Reply:
done